src/containers/genres/genres.html

```diff
@@ -0,0 +1,109 @@
+          <div class = "box">
+            <table class = "table">
+                <tr>
+                  <th><label class="checkbox"><input type="checkbox"> Comedy <i class="fas fa-laugh-squint"></i></label></th>
+                  <th><label class="checkbox"><input type="checkbox"> Crime <i class="fas fa-user-ninja"></i></label></th>
+                  <th><label class="checkbox"><input type="checkbox"> Thriller <i class="fas fa-crow"></i></label></th>
+                  <th><label class="checkbox"><input type="checkbox"> Adventure <i class="fas fa-map-signs"></i></label></th>
+                  <th><label class="checkbox"><input type="checkbox"> Sport <i class="fas fa-football-ball"></i></label></th>
+                </tr>       
+                <tr>
+                    <th><label class="checkbox"><input type="checkbox"> Family <i class="fas fa-home"></i></label></th>
+                    <th><label class="checkbox"><input type="checkbox"> Fantasy <i class="fas fa-dragon"></i></label></th>
+                    <th><label class="checkbox"><input type="checkbox"> Action <i class="fas fa-bomb"></i></label></th>
+                    <th><label class="checkbox"><input type="checkbox"> Detective <i class="fas fa-user-secret"></i></label></th>
+                    <th><label class="checkbox"><input type="checkbox"> Horror <i class="fas fa-ghost"></i></label></th>
+                  </tr>         
+                  <tr>
+                    <th><label class="checkbox"><input type="checkbox"> Romance <i class="fas fa-heart"></i></label></th>
+                    <th><label class="checkbox"><input type="checkbox"> Biography <i class="fas fa-user-tie"></i></label></th>
+                    <th><label class="checkbox"><input type="checkbox"> History <i class="fab fa-fort-awesome"></i></label></th>
+                    <th><label class="checkbox"><input type="checkbox"> Mystery <i class="fas fa-search"></i></label></th>
+                    <th><label class="checkbox"><input type="checkbox"> Musical <i class="fas fa-music"></i></label></th>
+                  </tr>         
+                  <tr>
+                    <th><label class="checkbox"><input type="checkbox"> Drama <i class="fas fa-theater-masks"></i></label></th>
+                    <th><label class="checkbox"><input type="checkbox"> Sci-Fi <i class="fas fa-robot"></i></label></th>
+                    <th><label class="checkbox"><input type="checkbox"> War <i class="fas fa-fighter-jet"></i></label></th>
+                    <th><label class="checkbox"><input type="checkbox"> Animation <i class="fab fa-optin-monster"></i></label></th>
+                    <th><label class="checkbox"><input type="checkbox"> Western <i class="fab fa-sticker-mule"></i></label></th>
+                  </tr>         
+              </table>
+        </div>
+      </div>
+
+    </div>
+
+      <button class="button is-large">BUTTON</button>
+
+</main>
+    
+</body>
+</html>
```

Comment:
Add script line to get nav bar automatically <script src="./src/containers/navigation-bar/js/nav.js"></script>